app/error_controller.go

```diff
@@ -0,0 +1,45 @@
+package app
+
+import (
+	"io"
+	"net/http"
+)
+
+func (app *App) Handler404Error(w http.ResponseWriter, r *http.Request) {
```

Comment:
why add it here?Why not have a middleware method?

Reply:
I found a inbuild function in bone with handle error so decided to go with it
to set it I had to write

`` app.Router.NotFoundFunc(app.renderView(app.Handler404Error))
and to access it

`` app.Router.HandleNotFound(w, r)

Reply:
you can write that in middleware so that everyone who needs this functionality can source it from there.
specifically check `response.go`